### ISSUE TITLE
Add support for account mail (RoDex)

### ISSUE THREAD

**Issue by Balferian** [rathena/rathena#6703]
**Is your request related to a missing feature?** 
yes
**Describe the solution you'd like**
**Describe alternatives you've considered**
**Additional context**
Neo make a patch that allow to use account mail tab:
https://github.com/Neo-Mind/WARP/issues/41
Be good to add support for account mail system.

**Comment by Lemongrass3110** [comment]
You can already do that via SQL insert if I remember correctly. ;-)

**Comment by Balferian** [comment]
with sql no problem, but isn`t better to make script command?

**Comment by Balferian** [comment]
Also accoun mail list should be loaded for all chars from account:
![image](https://user-images.githubusercontent.com/16213511/158828437-0f323483-7271-41f8-827e-d464074a6758.png)

and rA load it only for one character:
![image](https://user-images.githubusercontent.com/16213511/158828628-053c4207-e83d-4278-af20-8be5e5959693.png)

**Comment by Lemongrass3110** [comment]
Yeah. I never made one, since it was not supported by available game executables.
We can add support now that Neo added a diff. :-)

**Comment by Pokye** [comment]
Does this only work on Sakray clients?
